rplugin/python3/gdb/app.py

```diff
@@ -118,8 +121,7 @@ def breakpoint_toggle(self):
 
         if breaks:
             # There already is a breakpoint on this line: remove
-            del_br = self._get_command('delete_breakpoints')
-            self.client.send_line(f"{del_br} {breaks[-1]}")
+            self.debugger.delete_breakpoint(breaks[-1])
         else:
             set_br = self._get_command('breakpoint')
```

Comment:
Having changed `delete_breakpoint`, you'd better implement other commands the same way.